docs/src/cli/deploy-a-program.md

```diff
@@ -282,14 +282,24 @@ intermediary buffer contents and then vote to allow an upgrade using it.
 solana program write-buffer <PROGRAM_FILEPATH>
 ```
 
-Buffer accounts support authorities like program accounts:
+Buffer accounts are managed by some authority - default signer, or signer specified with 
+`--buffer-authority` option:
```

Comment:
We specify the use of a separate buffer-authority in a few other places, like close, and intro it as:

```suggestion
Buffer accounts are managed by an authority. To create a buffer and specify a different
authority than the default:
```

Reply:
Sure, applied in https://github.com/solana-labs/solana/pull/34366/commits/d35fbb099feb14301f9fb42e5acf252fa2af0a5b